tests/wot_coap_generation/config/wot_td/thing_model.json

```
@@ -5,14 +5,38 @@
     "description": "Lamp Thing Description Model",
     "properties": {
         "status": {
+            "title": "Light status",
+            "titles": {
+                "de": "Lampen status"
+            },
             "description": "current status of the lamp (on|off)",
+            "descriptions": {
+                "de": "Aktueller Lampen status"
+            },
             "type": "string",
             "readOnly": true
         }
     },
     "actions": {
         "toggle": {
-            "description": "Turn the lamp on or off"
+            "title": "Toggle Light",
+            "titles": {
+                "de": "Lampe ein/aus"
+            },
+            "description": "Turn the lamp on or off",
+            "descriptions": {
+                "de": "Schalte die Lampe aus und an"
+            }
+        },
+        "rmtoggle": {
+            "title": "Enable/Disable toggle",
+            "titles": {
+                "de": "aus/an"
+            },
+            "description": "Turns on/off the toggle endpoint",
+            "descriptions": {
+                "de": "Schaltet den toggle endpunkt aus/an"
+            }
```

This suggestion is not mandatory but recommended by the spec: https://w3c.github.io/wot-thing-description/#td-title-description

```suggestion
            "title": "Toggle light",
            "titles": {
                "de": "Lampe ein/aus",
                "en": "Toggle light"
            },
            "description": "Turn the lamp on or off",
            "descriptions": {
                "de": "Schalte die Lampe aus und an",
                "en": "Turn the lamp on or off"
            }
        },
        "rmtoggle": {
            "title": "Enable/disable toggle",
            "titles": {
                "de": "aus/an",
                "en": "Enable/disable toggle"
            },
            "description": "Turns on/off the toggle endpoint",
            "descriptions": {
                "de": "Schaltet den toggle-Endpunkt aus/an",
                "en": "Turns on/off the toggle endpoint"
            }
```